examples/01-simple-model/generated/binding.ts

```diff
@@ -211,11 +211,10 @@ export interface BaseModelUUID extends BaseGraphQLObject {
 }
 
 export interface PageInfo {
-  limit: Float
-  offset: Float
-  totalCount: Float
   hasNextPage: Boolean
   hasPreviousPage: Boolean
+  startCursor?: String | null
+  endCursor?: String | null
```

Comment:
Lots of these changes from the last PR.

Reply:
Interesting.  I think in your case since you're on 2.18.1 we should publish a 2.18.2 so that these changes don't come in.

Reply:
Actually if you're planning on jumping on 2.19 this should be fine.